server/startup/serverRunning.js

```diff
@@ -39,14 +52,20 @@ Meteor.startup(function() {
 
 		msg = msg.join('\n');
 
-		if (semver.satisfies(process.versions.node, desiredNodeVersionMajor)) {
-			return SystemLogger.startup_box(msg, 'SERVER RUNNING');
+		if (!semver.satisfies(process.versions.node, desiredNodeVersionMajor)) {
+			msg += ['', '', 'YOUR CURRENT NODEJS VERSION IS NOT SUPPORTED,', `PLEASE UPGRADE / DOWNGRADE TO VERSION ${ desiredNodeVersionMajor }.X.X`].join('\n');
+			SystemLogger.error_box(msg, 'SERVER ERROR');
+
+			return process.exit();
```

Comment:
should we call this as `[BREAK]` because of this? like, people will need to know before hand if they upgrade and are using a not desired node or mongo version the server will not start.

Reply:
I’ll change title.  I think that’s fair.  But it actually wouldn’t work anyways before it was leading to tickets open.

Better to do this as a breaking change now with 1.0 then after